packages/admin/admin/src/form/Autocomplete.tsx

```diff
@@ -56,8 +59,10 @@ export const FinalFormAutocomplete = <
                                 {clearable && (
                                     <ClearInputAdornment position="end" hasClearableContent={Boolean(value)} onClick={() => onChange("")} />
                                 )}
-                                {params.InputProps.endAdornment}
+                                {endAdornment ? endAdornment : params.InputProps.endAdornment}
```

Comment:
Why can't we simply use `params.InputProps.endAdornment`? Wouldn't it be enough to add a story that shows how to set the end adornment? @jamesricky what do you think?